docs/en/community/team.md

```diff
@@ -24,62 +24,60 @@ specific language governing permissions and limitations
 
-## Committers (33)
+## Committers (38)
 
-(the listing excludes PPMC members above)
+(the listing excludes PMC members above)
 
-| Apache ID                                                    | Github Username  | Public Name    |
+| Apache ID                                                    | Github Username    | Public Name         |
 | :----------------------------------------------------------- | :--------------- | :------------- |
+| [lichaoyong](https://people.apache.org/committer-index.html#lichaoyong) | chaoyli             | Chaoyong Li   |
+| [zhaoc](https://people.apache.org/committer-index.html#zhaoc) | imay                | Chun Zhao     |
 | [diwu](https://people.apache.org/committer-index.html#diwu)  | JNSimba          | Di Wu          |
 | [yiguolei](https://people.apache.org/committer-index.html#yiguolei) | yiguolei         | Guolei Yi      |
+| [liuhangyuan](https://people.apache.org/committer-index.html#liuhangyuan) | HangyuanLiu         | Hangyuan Liu  |
 | [huajianlan](https://people.apache.org/committer-index.html#huajianlan) | 924060929        | Huajian Lan    |
 | [huangwei](https://people.apache.org/committer-index.html#huangwei) | vagetablechicken | Huang Wei      |
 | [jianliangqi](https://people.apache.org/committer-index.html#jianliangqi) | qidaye           | Jianliang Qi   |
+| [kangkaisen](https://people.apache.org/committer-index.html#kangkaisen) | kangkaisen          | Kaisen Kang   |
 | [kangpinghuang](https://people.apache.org/committer-index.html#kangpinghuang) | kangpinghuang    | Kangping Huang |
 | [liulijia](https://people.apache.org/committer-index.html#liulijia) | liutang123       | Lijia Liu      |
+| [panxiaolei](https://people.apache.org/committer-index.html#panxiaolei) | BiteTheDDDDt       | Xiaolei Pan      |
+| [shaofengshi](https://people.apache.org/committer-index.html#shaofengshi) | shaofengshi     | Shao Feng Shi     |
+| [sijie](https://people.apache.org/committer-index.html#sijie) | sijie               | Sijie Guo     |
+| [weixiang](https://people.apache.org/committer-index.html#weixiang) | spaces-X            | Xiang Wei     |
+| [zouxinyi](https://people.apache.org/committer-index.html#zouxinyi) | xinyiZzz               | Xinyi Zou     |
 | [wangxixu](https://people.apache.org/committer-index.html#wangxixu) | xinghuayu007     | Xixu Wang      |
 | [xuyang](https://people.apache.org/committer-index.html#xuyang) | xy720            | Yang Xu        |
 | [laiyingchun](https://people.apache.org/committer-index.html#laiyingchun) | acelyc111        | Yingchun Lai   |
+| [dataroaring](https://people.apache.org/committer-index.html#dataroaring) | dataroaring | Yongqiang Yang   |
 | [wyf](https://people.apache.org/committer-index.html#wyf)    | wuyunfeng        | Yunfeng Wu     |
+| [zenoyang](https://people.apache.org/committer-index.html#zenoyang)    |         | Zeno Yang     |
+| [zshao](https://people.apache.org/committer-index.html#zshao)    | wuyunfeng        | Yunfeng Wu     |
```

Comment:
| [zshao](https://people.apache.org/committer-index.html#zshao)    | wuyunfeng        | Yunfeng Wu     |   
is this right

Reply:
fix